### PR TITLE
#28: Restructure code.

### PR DESCRIPTION
**Pull request by salihgueler**
Part of #28

### CODE REVIEW

File: lib/flutter_festival_home_page.dart [hunk 1]
```diff
@@ -1,18 +1,19 @@
...
-import 'package:hackathon_flutterturkiye_org/widgets/footer_view.dart';
-import 'package:hackathon_flutterturkiye_org/widgets/section_title.dart';
-import 'package:hackathon_flutterturkiye_org/widgets/speakers_section.dart';
+import 'package:hackathon_flutterturkiye_org/core/ui/section_title.dart';
+import 'package:hackathon_flutterturkiye_org/footer/ui/footer_view.dart';
+import 'package:hackathon_flutterturkiye_org/speaker/ui/speakers_section.dart';
 
-class HomePage extends StatefulWidget {
-  HomePage({Key key, this.title}) : super(key: key);
+class FlutterFestivalHomePage extends StatefulWidget {
```
Comment (bgoktugozdemir) [inline lib/flutter_festival_home_page.dart:6]: Why we changed the name of Home Page? Will we add another page?

**Review by bgoktugozdemir — APPROVED** [review]

**Review by mukireus — COMMENTED** [review]
looks good to me

**Review by mukireus — APPROVED** [review]
looks good to me

**Comment by Adem68** [comment]
lgtm

**Review by Adem68 — APPROVED** [review]

File: lib/flutter_festival_home_page.dart (hunk 1 above)
Comment (salihgueler) [reply lib/flutter_festival_home_page.dart:6]: Because it will be a single page web page :)

Comment (bgoktugozdemir) [reply lib/flutter_festival_home_page.dart:6]: If it is a single web page, Why, we changed the name of it? It can be HomePage.

**Comment by salihgueler** [comment]
Reimplementing this would be simpler.

**Pull request closed without merging**